### PR TITLE
[thread-cert] refactor case 9.2.2 using pktverify

### PR DESCRIPTION
**Pull request by FiveDimensions**

### CODE REVIEW

File: tests/scripts/thread-cert/Cert_9_2_02_MGMTCommissionerSet.py [hunk 1]
```diff
@@ -65,126 +84,308 @@ def test(self):
         self.nodes[COMMISSIONER].start()
         self.simulator.go(5)
         self.assertEqual(self.nodes[COMMISSIONER].get_state(), 'router')
-
-        # Skip all other Coaps sent by Leader
-        self.simulator.get_messages_sent_by(COMMISSIONER)
         self.simulator.get_messages_sent_by(LEADER)
 
-        # Commissioner start
+        self.collect_leader_aloc(LEADER)
```
Comment (simonlingoogle) [inline tests/scripts/thread-cert/Cert_9_2_02_MGMTCommissionerSet.py:89]: nit: Leader ALOC is fixed in tests

File: tests/scripts/thread-cert/Cert_9_2_02_MGMTCommissionerSet.py [hunk 2]
```diff
@@ -65,126 +84,308 @@ def test(self):
...
+        LEADER = pv.vars['LEADER']
+        LEADER_ALOC = pv.vars['LEADER_ALOC']
+        LEADER_RLOC = pv.vars['LEADER_RLOC']
+        LEADER_RLOC16 = pv.vars['LEADER_RLOC16']
+        COMMISSIONER = pv.vars['COMMISSIONER']
+        COMMISSIONER_RLOC = pv.vars['COMMISSIONER_RLOC']
+
+        # Step 1: Ensure topology is formed correctly
+        pv.verify_attached('COMMISSIONER', 'LEADER')
+        _pkt = pkts.last()
```
Comment (simonlingoogle) [inline tests/scripts/thread-cert/Cert_9_2_02_MGMTCommissionerSet.py:149]:
```suggestion
```
?

**Review by simonlingoogle — APPROVED** [review]
LGTM :+1:

File: tests/scripts/thread-cert/Cert_9_2_02_MGMTCommissionerSet.py (hunk 2 above)
Comment (FiveDimensions) [reply tests/scripts/thread-cert/Cert_9_2_02_MGMTCommissionerSet.py]: deleted

File: tests/scripts/thread-cert/Cert_9_2_02_MGMTCommissionerSet.py (hunk 1 above)
Comment (FiveDimensions) [reply tests/scripts/thread-cert/Cert_9_2_02_MGMTCommissionerSet.py]: add const

**Review by jwhui — APPROVED** [review]

**Pull request merged**